input/docs/handbook/dependency-inversion/custom-dependency-inversion.md

```diff
@@ -0,0 +1,67 @@
+# Override Default Depenedency Inversion Container
+
+We understand that some developers would prefer to use their favorite dependency inversion container.  ReactiveUI allows for this by implementing `IMutableDependencyResolver`.  Once this class has been implemented you simply need to set it to the `Locator.
+
+## Implement an `IMuatableDependencyResolver`
+
+```csharp
+public abstract class MyDependencyResolver : IMutableDependencyResolver
+{
+    public abstract object GetService(Type serviceType, string contract = null);
+
+    public abstract IEnumerable<object> GetServices(Type serviceType, string contract = null);
+
+    public abstract void Register(Func<object> factory, Type serviceType, string contract = null);
+
+    public abstract IDisposable ServiceRegistrationCallback(Type serviceType, string contract, Action<IDisposable> callback);
+
+    public abstract void Dispose();
+}
+```
+
+## Register services against `MyDepedencyResolver`
+
+```csharp
+public class MyDepedencyResolver
+{
+    protected IMyContainer Container;
+
+    public MyDepedencyResolver(IMyContainer container)
+    {
```

Comment:
Potentially you might want to talk about InitializeReactiveUI which will re-register all the IPlatformsRegister stuff into the new container.

Reply:
Adding something here is fine.  There needs to be some verbiage on the site about those methods and what they do, which I think is beyond the scope of this page.